### PR TITLE
Convert bash pipeline scripts to PERL.

### PR DESCRIPTION
**Pull request by davelcpanelnet**
There were 3 scripts originally done in BASH that i wanted to convert over to PERL for more flexibility and control.

### CODE REVIEW

**Comment by davelcpanelnet** [comment]
https://github.com/cpanel/elevate/actions/runs/10693725730/job/29645442689 show this one passing.

File: .github/workflows/openstack.yml [hunk 1]
```diff
@@ -150,10 +150,12 @@ jobs:
             username: 'root'
             key: ${{ secrets.SSH_PRIVATE_KEY }}
             port: '22'
+            timeout: 30m
+            command_timeout: 30m
             script: |
               cp -pv /opt/elevate/elevate-cpanel /scripts/elevate-cpanel
-              cp -pv /opt/elevate/.github/workflows/openstack/status_marker.sh /scripts/status_marker.sh
-              cp -pv /opt/elevate/.github/workflows/openstack/reboot_watch.sh /scripts/reboot_watch.sh
+              cp -pv /opt/elevate/.github/workflows/openstack/status_marker.pl /scripts/status_marker.pl
```
Comment (toddr) [inline .github/workflows/openstack.yml:157]: Why end in pl or sh? That's not a relevant detail and just leads to more change like here I suggest stripping the extension

**Review by toddr — CHANGES_REQUESTED** [review]

**Comment by davelcpanelnet** [comment]
In my head only compiled binaries get no extension.

happy to make that change.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: .github/workflows/openstack.yml (hunk 1 above)
Comment (davelcpanelnet) [reply .github/workflows/openstack.yml]: Latest commit should have these removed.

**Review by cPholloway — CHANGES_REQUESTED** [review]
Please squash the commits and update provide a meaningful commit message

**Review by cPholloway — APPROVED** [review]

**Review by toddr — APPROVED** [review]

**Pull request merged**